common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -454,6 +456,27 @@ def copy_from_published(root_block_id):
             if index_entry is not None:
                 self._update_head(draft_course_key, index_entry, ModuleStoreEnum.BranchName.draft, new_structure['_id'])
 
+    def update_parent_if_moved(self, item_location, original_parent_location, course_structure, user_id):
+        """
+        Update parent of an item if it has moved.
+
+        Arguments:
+            item_location (BlockUsageLocator)    : Locator of item.
+            original_parent_location (BlockUsageLocator)  : Original parent block locator.
+            course_structure (Structure)  : course structure of the course.
+            user_id (int)   : User id
+        """
+        parent_block_keys = self._get_parents_from_structure(BlockKey.from_usage_key(item_location), course_structure)
+        for block_key in parent_block_keys:
```

Comment:
If the code looks for the parent of `item_location`, in what case would it return more than one parent? Is this code to guard against items having more than one parent (directed acyclic graph)?